### PR TITLE
dotCMS/core#18021 Container not behaving correctly when reaching max contents

### PR DESCRIPTION
**Pull request by hmoreras**

### CODE REVIEW

File: src/app/portlets/dot-edit-page/content/shared/iframe-edit-mode.css.ts [hunk 1]
```diff
@@ -231,11 +235,19 @@ export const getEditPageCss = (timestampId: string): string => {
...
-    ${timestampId} .dotedit-menu__item[title]:hover:after {
-        content: attr(title) !important;
-        left: 100% !important;
+    ${timestampId} .dotedit-menu__item[dot-title]:hover:after {
+        content: attr(dot-title) !important;
+        right: 100% !important;
         position: absolute !important;
-        top: 100% !important;
+        top: 4px !important;
+        background: rgba(0,0,0,.7);
```
Comment (alfredo-dotcms) [inline src/app/portlets/dot-edit-page/content/shared/iframe-edit-mode.css.ts:243]: are these styles only for these specific tooltip or should it be move to a general place, so all tooltips look the same?

File: src/app/portlets/dot-edit-page/content/services/html/dot-edit-content-toolbar-html.service.ts [hunk 2]
```diff
@@ -301,7 +301,7 @@ export class DotEditContentToolbarHtmlService {
                             <li class="dotedit-menu__item ${
                                 item.disabled ? 'dotedit-menu__item--disabled' : ''
                             }"
-                                ${item.tooltip ? 'title="' + item.tooltip + '"' : ''}">
+                                ${item.tooltip ? 'dot-title="' + item.tooltip + '"' : ''}">
```
Comment (freddyucv) [inline src/app/portlets/dot-edit-page/content/services/html/dot-edit-content-toolbar-html.service.ts:304]: test?

File: src/app/portlets/dot-edit-page/content/shared/iframe-edit-mode.css.ts (hunk 1 above)
Comment (hmoreras) [reply src/app/portlets/dot-edit-page/content/shared/iframe-edit-mode.css.ts:243]: just specific for this scenario.

File: src/app/portlets/dot-edit-page/content/services/html/dot-edit-content-toolbar-html.service.ts (hunk 2 above)
Comment (hmoreras) [reply src/app/portlets/dot-edit-page/content/services/html/dot-edit-content-toolbar-html.service.ts:304]: done

**Pull request merged**